rules/ephemeral/generator/main.go

```diff
@@ -0,0 +1,68 @@
+//go:build generators
```

Comment:
This is not necessary since the generator is a different package. When `go build` runs, it walks the graph of all `import` statements found in the package for which it was invoked, plus transitive dependencies. This generator shouldn't be anywhere in that graph, since it never actually gets imported, only invoked via `go run`. You'd only need a build tag on this file if the package were imported in order to use things exported from other files.